clang/lib/Sema/SemaTemplateInstantiateDecl.cpp

```diff
@@ -4086,7 +4073,7 @@ FunctionDecl *Sema::SubstSpaceshipAsEqualEqual(CXXRecordDecl *RD,
   Decl *R;
   if (auto *MD = dyn_cast<CXXMethodDecl>(Spaceship)) {
     R = Instantiator.VisitCXXMethodDecl(
-        MD, nullptr, std::nullopt,
+        MD, nullptr,
```

Comment:
```suggestion
        MD, /*TemplateParams=*/nullptr,
```

Reply:
To be consistent with [bugprone-argument-comment](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html)